src/main/webapp/app/overview/exercise-details/course-exercise-details.component.html

```diff
@@ -251,4 +251,5 @@ <h3>{{ 'artemisApp.courseOverview.exerciseDetails.allResults' | artemisTranslate
         </div>
     </div>
     <jhi-lti-initializer></jhi-lti-initializer>
+    <jhi-exercise-chatbot></jhi-exercise-chatbot>
```

Comment:
If you add the button here, you'll have to make sure that it is only displayed when the exercise is a programming exercise, since this is the only type of excercise we want to support for now.

Reply:
Why can't you create a programming exercise? 
For this exact reason it is essentail that you have a working local setup.

Reply:
We tried to install it with the Intructor for 1 hour extra and it kept on giving Internal Server Error. I think there is not more I can do at this point. Maybe Asli would be able to create for herself. But then you will have to wait for the basic UI representation.

Reply:
@eceeeren
[Chatbot made available only for programming](https://github.com/ls1intum/Artemis/pull/6584/commits/976390a9cc8ad1ca9032e89e890d82203993f518)